README.md

```diff
@@ -5,9 +5,14 @@ A quiz that allows people to test how well their perception of people's online i
 
 Before cloning the application, install the Gulp CLI globally by running `npm install gulp-cli -g`. If you're using nvm you can run `nvm use` to run the version of Node this project currently supports. Otherwise refer to the `package.json` file in the root and install and switch to the correct version of Node.
 
+You also need mongodb. Assuming you have homebrew, run `brew install mongo`.
+
 ## Install Instructions
 
-To install, clone the repository by selecting the clone or download button from the [Github Repository](https://github.com/BuildBuds/who-are-you). Once the repository has been cloned, change directories to the repositories root and run `npm install`. Once the application is installed run `npm run start` or `gulp` to start the application. The application should be up and running and will be proxied to `http://localhost:3000` by browser-sync.
+To install, clone the repository by selecting the clone or download button from the [Github Repository](https://github.com/BuildBuds/who-are-you). Once the repository has been cloned, change directories to the repositories root and run `npm install`.
+
+## Running locally
+Once the application is installed, open another terminal session and run `mongod`. Then, run `npm run start` or `gulp` to start the application. The application should be up and running and will be proxied to `http://localhost:8080` by browser-sync.
```

Comment:
I can only get this to work if I run `sudo mongod` based on the mongo db's install instructions. Are there ways around this?

Reply:
that's interesting. i can run it fine without being super user. this looks like it answers your question https://stackoverflow.com/questions/5300861/mongodb-only-works-when-run-as-root-on-ubuntu-data-directory-issue